osu.Game/Screens/Edit/Timing/MetronomeDisplay.cs

```diff
@@ -248,14 +252,27 @@ protected override void Update()
             if (BeatSyncSource.Clock?.IsRunning != true && isSwinging)
             {
                 swing.ClearTransforms(true);
+                stick.FadeColour(overlayColourProvider.Colour2, 1000, Easing.OutQuint);
+                isSwinging = false;
+
+                // instantly latch if pendulum arm is close enough to center (to prevent awkward delayed playback of latch sound)
+                if (Precision.AlmostEquals(swing.Rotation, 0, 1))
+                {
+                    swing.RotateTo(0);
+                    latch?.Play();
+                    return;
+                }
 
                 using (swing.BeginDelayedSequence(350))
                 {
                     swing.RotateTo(0, 1000, Easing.OutQuint);
-                    stick.FadeColour(overlayColourProvider.Colour2, 1000, Easing.OutQuint);
```

Comment:
Might be only me, but I still prefer delaying the colour fade to give visual feedback about the played metronome tick when pausing the clock.

master:

https://user-images.githubusercontent.com/22781491/174319638-e7121da6-0ca3-473a-a7f0-5eaf035b9739.mp4

PR:

https://user-images.githubusercontent.com/22781491/174319591-aaa387c7-5ee9-49de-a683-b85f540f67e7.mp4

Reply:
Definitely prefer how it was previously as that's how I implemented it 🙃

Reply:
Have reverted for now. Can understand why it was changed but want to play with it with the old effect and see if it disturbs me after some time using it.